lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -0,0 +1,138 @@
+"""
+Tests for credit requirement display on the progress page.
+"""
+
+import datetime
+
+from mock import patch
+from pytz import UTC
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+from student.tests.factories import UserFactory, CourseEnrollmentFactory
+from util.date_utils import get_time_display, DEFAULT_SHORT_DATE_FORMAT
+
+from openedx.core.djangoapps.credit import api as credit_api
+from openedx.core.djangoapps.credit.models import CreditCourse
```

Comment:
This really is a nitpick, but I typically don't break those groups up internally with newlines. PEP 8 only recommends putting a blank line between each group of imports. I'd turn this into:
```python
import datetime

from django.conf import settings
from django.core.urlresolvers import reverse
from mock import patch
from pytz import UTC

from openedx.core.djangoapps.credit import api as credit_api
from openedx.core.djangoapps.credit.models import CreditCourse
from student.tests.factories import UserFactory, CourseEnrollmentFactory
from util.date_utils import get_time_display, DEFAULT_SHORT_DATE_FORMAT
from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
from xmodule.modulestore.tests.factories import CourseFactory
```